shopify/usage-subscription-template/api/actions/billingPeriodTracking.ts

```diff
@@ -60,13 +55,13 @@ export async function run({ params, logger, api, connections }) {
           activeSubscriptionId: shop.activeSubscriptionId,
           usagePlanId: shop.usagePlanId,
           plan: {
-            price: shop.plan.pricePerOrder,
+            price: shop.plan ? shop.plan.pricePerOrder : 0,
           },
         },
       },
       {
         queue: {
-          name: shop.name,
+          name: shop.name as string,
```

Comment:
We can probably add a falsy check for shop.name so we don't have to cast as string